aws_copilot_deploy/aws-api.Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM python:3.11
```

Comment:
Thank you for putting all these in a separate folder, great work

Reply:
Thanks